java/ql/test/library-tests/frameworks/apache-commons-lang3/TripleTest.java

```diff
@@ -0,0 +1,130 @@
+import org.apache.commons.lang3.tuple.Triple;
+import org.apache.commons.lang3.tuple.ImmutableTriple;
+import org.apache.commons.lang3.tuple.MutableTriple;
+
+class TripleTest {
+    String taint() { return "tainted"; }
+
+    private static class IntSource {
+      static int taint() { return 0; }
+    }
+
+    void sink(Object o) {}
+
+    void test() throws Exception {
```

Comment:
This seems to be missing a test for `Triple.of(left, middle, right)`
Might be good to add it for completeness?